### PR TITLE
Confidenceman02 reactions/content card

### PR DESCRIPTION
**Pull request by Confidenceman02**
- added static reaction card
- added dialogue box
- added icon
screen shot 2017-12-02 at 6 00 27 pm

### CODE REVIEW

File: src/components/DialogueBox/DialogueBox.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import './_utils.scss';
+@import '../../common/_variables.scss';
+
+.tip_left {
```
Comment (rodleviton) [inline src/components/DialogueBox/DialogueBox.scss]: a small thing but we need to agree on naming e.g. `_` v `-` v camelCase.

File: src/components/Stage/index.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
 import React, { Component } from 'react'
```
Comment (rodleviton) [inline src/components/Stage/index.js]: I will need to move the codeEditor into this stage format.

**Comment by rodleviton** [comment]
Apart from the lame comment placeholder text this looks very cool.

File: src/components/DialogueBox/DialogueBox.scss (hunk 1 above)
Comment (Confidenceman02) [reply src/components/DialogueBox/DialogueBox.scss]: This is just a variables partial for this particular component.

**Pull request merged**